telepresence/proxy/remote.py

```diff
@@ -28,27 +30,30 @@ class RemoteInfo(object):
     :ivar container_config dict: The container within the Deployment JSON.
     :ivar container_name str: The name of the container.
     """
-    def __init__(
-        self,
-        runner: Runner,
-        deployment_name: str,
-        pod_name: str,
-        deployment_config: dict,
-    ) -> None:
+    def __init__(self, pod_name: str, pod_spec: Manifest) -> None:
```

Comment:
Given what this function does, how likely is it that `pod_spec` is ever written again? And will the pod spec ever have more than the telepresence container? If not, would it make sense to capture tel_container as an instance variable and skip retaining `_container_index`?

Also, maybe just accept the pod rather than requiring the caller to pull out the name and spec themselves?